stdlib/internal/src/main/ballerina/internal/jwt_validator.bal

```diff
@@ -213,8 +215,8 @@ function validateMandatoryFields(JwtPayload jwtPayload) returns (boolean) {
     return true;
 }
 
-function validateSignature(string[] encodedJWTComponents, JwtHeader jwtHeader, JWTValidatorConfig config) returns (
-        boolean) {
+function validateSignature(string[] encodedJWTComponents, JwtHeader jwtHeader, JWTValidatorConfig config) returns
```

Comment:
Shall we move returns word below along with error?

Reply:
Done. Thanks!